src/app/model/sell/sell.ts

```diff
@@ -0,0 +1,19 @@
+export interface Sell {
+    customerInformation: CustomerInformation,
+    totalAmount: number
+    cacke: CackeSold;
+}
+
+interface CustomerInformation {
+    customerName: string;
+    customerPhoneNumber: string;
+    customerEmail: string;
+}
+
+interface CackeSold  {
+    name: string
+    descrition: string
+    ingredients: string[]
+    price: number
+    quantity: number
+}
```

Comment:
On this challenge we are following a single responsibility file, then you should separate all the elements by file.